GoogleDataTransport/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v3.0.0
+- Changes backgrounding logic to reduce background usage and properly complete
+all tasks. (#3893)
+
```

Comment:
Add
-     Fix Catalyst define checks (#3695 
-     Fix ubsan issues in GDT (#3910)
-     Add support for FLL (Firelog ?????) (#3867)

Reply:
Didn't realize all those changed, thanks for bringing it up!